lib/registration.pm

```diff
@@ -459,6 +459,11 @@ sub get_addon_fullname {
 
 
 sub fill_in_reg_server {
+    # Set product specific SCC_REGCODE if it was provided. Defaults to SCC_REGCODE if set
+    my $scc_code = get_var('SCC_REGCODE', '') if get_var('SCC_REGCODE');
+    $scc_code = get_var('SCC_REGCODE_' . uc(get_var('SLE_PRODUCT')), '')
```

Comment:
please assign get_var('SCC_REGCODE_' . uc(get_var('SLE_PRODUCT')), '') to another variable and do not duplicate the construct.